src/views/AddTenant/addTenant.js

```diff
@@ -158,8 +178,13 @@ export const AddTenant = () => {
             occupants: "",
           }}
           validationSchema={validationSchema}
+          validate={() => {
+            setIsValidationActive(true);
+            validatePropertySelection();
+            validateLeaseDates();
```

Comment:
We probably only want to validate this if there is any data on these two forms; otherwise only validate the tenant form. Although, I'm not sure if that is what this is doing already?

Reply:
Ok. Now that I have a much better understanding of the backend .... the form will validate propertyID if lease dates are supplied. Lease dates are optional though, so propertyID is not required until after the lease dates are entered.